packages/assistify-help-request/server/models/Rooms.js

```diff
@@ -36,5 +36,25 @@ _.extend(RocketChat.models.Rooms, {
 			{$or};
 
 		return this.find(query, options);
+	},
+
+	/**
+	 * Find all public accessible rooms where the given user is not subscribed to
+	 *
+	 * @param {string} username - the username which should be excluded
+	 * @param options
+	 */
+	findAllPublicNotContainingUserName(username, options) {
+		const query = {
+			t: {
+				//todo are those types all public accesible ones?
+				$in: ['r', 'l', 'c']
```

Comment:
@lobo87 I don't think this works out. `r` and `l` are not public. Those should be either found using the subscriptions or there could be a "joined rooms" query instead

Reply:
@mrsimpson I already have the subscriptions, this method is intended to fetch all accessible public rooms, where the user has NOT joined.